homepage/tests/course_tests.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.parametrize("valid_course_id", [
+    (10231), (10111), (10221), (12357)
+    ])
+@pytest.mark.django_db
+def test_renders_course_template(client, valid_course_id):
+    response = client.get('/course/' + str(valid_course_id) + '/')
+

```

Comment:
Kobi suggested in previous reviews that we remove the blank lines before the 'assert', so i think it needs to be removed here as well

Reply:
I had to make another update so I fixed it on the way :)